tf-modules/log-subscription-filter/main.tf

```diff
@@ -6,7 +6,3 @@ terraform {
     }
   }
 }
-
-locals {
-  destination_arn = var.log_destination_arn != null ? var.log_destination_arn : var.log2elasticsearch_lambda_function_arn
```

Comment:
So what happens when `var.log_destination_arn` is null now? Does the deployment blow up?

Reply:
The module won't even be deployed if there is no `log_destination_arn`. See: https://github.com/nasa/cumulus/blob/012de8a87ed6cfc4cfc98b13fc7151c6dcfdbca3/tf-modules/cumulus/log_subscription_filter.tf#L2